daisy_workflows/image_import/ubuntu/translate.py

```diff
@@ -160,6 +159,20 @@ def DistroSpecific(g):
   g.command(['update-grub2'])
 
 
+def remove_azure_agents(g):
```

Comment:
Isn't it most likely that Azure agent won't be present, unless an image is actually from Azure? Will the warning message show in that case as well? If so, we should make it sound less serious.

Reply:
That's a very good point. We're already logging the apt message at a debug level, which will just say the package wasn't found. So I just removed the warn message.